tools/mirror.toit

```diff
@@ -478,26 +479,24 @@ class MallocReport extends Mirror:
       result.add "│$(%2d k)k pages.  All pages are $(%2d k)k, even the ones that are shown wider       │"
       result.add "│ because they have many different allocations in them.                  │"
     else:
-      result.add "│Each line is a $(%2d k)k page.                                                      │"
-    result.add   "│   X  = External strings/bytearrays.        B  = Network buffers.       │"
-    result.add   "│   W  = TLS/crypto.                         M  = Misc. allocations.     │"
-    result.add   "│   🐱 = Toit managed heap.                  -- = Free page.             │"
+      result.add "│Each line is a $(%2d k)k page.                                                │"
     if terminal:
+      result.add "│   X  = External strings/bytearrays.        B  = Network buffers.       │"
+      result.add "│   W  = TLS/crypto.                         M  = Misc. allocations.     │"
+      result.add "│   🐱 = Toit managed heap.                  -- = Free page.             │"
       result.add "│        Fully allocated $scale Completely free page.  │"
     result.add   "└────────────────────────────────────────────────────────────────────────┘"
 
   usage_letters_ use/int fullness/int -> string:
-    symbols := ""
-    if use & MEMORY_PAGE_TOIT_ != 0: symbols += "🐱"
-    if use & MEMORY_PAGE_BUFFERS_ != 0: symbols = "B"
-    if use & MEMORY_PAGE_EXTERNAL_ != 0: symbols = "X"
-    if use & MEMORY_PAGE_TLS_ != 0: symbols = "W"
-    if use & MEMORY_PAGE_MISC_ != 0: symbols = "M"
+    symbols := []
```

Comment:
I don't understand this change.
Doesn't the call-site need to change as well?

Reply:
Renamed to make it more clear that it's only used for plain ASCII mode, where the exact layout is not so constrained.